redis/cluster.py

```diff
@@ -2163,6 +2162,8 @@ def _send_cluster_commands(
                 if n.connection:
                     n.connection.disconnect()
                     n.connection_pool.release(n.connection)
+            if len(nodes) > 0:
```

Comment:
Just for curious, Is there any reason for delaying after the disconnect the connections?

Reply:
this code is a safety feature to catch all connection leaks, but it comes with a cost of disconnecting all connections used in this function. Ideally, this code block shouldn't be executed, but if executed due to an unknown error, it means that there probably is a problem with Soda or Redis. Reconnecting can put a burden more on a Redis node in such a case, so i thought static backoff could alleviate the load on Redis CPU